store/tools/ics23-smt/Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: build test testgen
```

Comment:
any chance we can combine make files, one command to test both iavl & smt and then individually

Reply:
Sure